index.js

```diff
@@ -684,10 +684,16 @@ function Runtime() {
                 return method;
 
             const tokens = parseMethodName(rawName);
+            const fullName = tokens[2];
+
+            method = cachedMethods[fullName];
+            if (method !== undefined) {
+              return method;
```

Comment:
- Shouldn't we add the `rawName` here so we can avoid the parsing on the next call?
- We still use 4 spaces in this module, so we should stick to that for now. (I'd like to reformat everything at some point.)